adam-python/bdgenomics/adam/rdd.py

```diff
@@ -692,6 +692,11 @@ def _inferConversionFn(self, destClass):
 class FeatureRDD(GenomicDataset):
 
 
+    def _replaceRdd(self, newRdd):
```

Comment:
Is this only necessary in `class FeatureRDD`?

Reply:
@heuermh if it is not present transform will fail (_replaceRdd is not a function)

Reply:
https://github.com/akmorrow13/adam/blob/eec00a545dca3e4cd4e258023c99a1afa8a8974e/adam-python/bdgenomics/adam/rdd.py#L104 is defined in GenomicRDD, calling _replaceRDD, which FeatureRDD does not have

Reply:
Thanks!